conans/client/rest/rest_client_common.py

```diff
@@ -22,24 +22,19 @@ def __call__(self, request):
         return request
 
 
-def _base_error(error_code):
-    return int(str(error_code)[0] + "00")
-
-
 def get_exception_from_error(error_code):
-    try:
-        tmp = {}
-        for key, value in EXCEPTION_CODE_MAPPING.items():
-            if key not in (RecipeNotFoundException, PackageNotFoundException):
-                tmp[value] = key
-        if error_code in tmp:
-            logger.debug("REST ERROR: %s" % str(tmp[error_code]))
-            return tmp[error_code]
-        else:
-            logger.debug("REST ERROR: %s" % str(_base_error(error_code)))
-            return tmp[_base_error(error_code)]
-    except KeyError:
-        return None
+    tmp = {v: k for k, v in EXCEPTION_CODE_MAPPING.items()  # All except NotFound
+           if k not in (RecipeNotFoundException, PackageNotFoundException)}
+    if error_code in tmp:
+        logger.debug("REST ERROR: %s" % str(tmp[error_code]))
+        return tmp[error_code]
+    else:
+        base_error = int(str(error_code)[0] + "00")
+        logger.debug("REST ERROR: %s" % str(base_error))
+        try:
```

Comment:
This should cover both return lines to catch any exiting KeyError

Reply:
All potential Key errors are covered here.  The above is conditioned to ``if error_code in tmp``